bootstrap.d/dev-libs.yml

```diff
@@ -903,6 +903,46 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: libxslt
+    source:
+      subdir: ports
+      git: 'https://gitlab.gnome.org/GNOME/libxslt.git'
+      tag: 'v1.1.34'
+      version: '1.1.34'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-libtool
+        - host-pkg-config
+      regenerate:
+        - args: ['./autogen.sh']
+          environ:
+            'NOCONFIGURE': 'yes'
+    tools_required:
+      - system-gcc
+    pkgs_required:
+      - mlibc
+      - libxml
+    configure:
+      - args: ['sed', '-i', 's/3000/5000/', '@THIS_SOURCE_DIR@/libxslt/transform.c']
+      - args: ['sed', '-i', 's/3000/5000/', '@THIS_SOURCE_DIR@/doc/xsltproc.1'] 
+      - args: ['sed', '-i', 's/3000/5000/', '@THIS_SOURCE_DIR@/doc/xsltproc.xml']
```

Comment:
That does this do?

Reply:
Uhh, taken from BLFS. I'll add a comment.

Reply:
This has been addressed.